spring-amqp/src/main/java/org/springframework/amqp/utils/JavaUtils.java

```diff
@@ -66,4 +69,68 @@ private JavaUtils() {
 		return this;
 	}
 
+	/**
+	 * Invoke {@link Consumer#accept(Object)} with the value if it is not null or empty.
+	 * @param value the value.
+	 * @param consumer the consumer.
+	 * @return this.
+	 */
+	public JavaUtils acceptIfHasText(String value, Consumer<String> consumer) {
+		if (StringUtils.hasText(value)) {
+			consumer.accept(value);
+		}
+		return this;
+	}
+
+	/**
+	 * Invoke {@link BiConsumer#accept(Object, Object)} with the arguments if the
+	 * condition is true.
+	 * @param condition the condition.
+	 * @param value the second consumer argument
+	 * @param t the first consumer argument
```

Comment:
The new Spring Check Style doesn't like single-letter variables.
Maybe it would be better to work out a good manner for us in our projects as well.
Plus it isn't clear why you chave this order for arguments while it is much natural to have the "first" as first and "second" as second. I mean how about to swap `value` and `t` in the method contract?